users/talljoe/rules.mk

```diff
@@ -0,0 +1,7 @@
+SRC += talljoe.c
+
+ifneq ("$(wildcard users/$(KEYMAP)/config.h)","")
```

Comment:
This isn't needed. It's part of the main build script now: https://github.com/qmk/qmk_firmware/blob/master/build_keyboard.mk#L219-L221